### PR TITLE
R4.3.1 + dev-modulefixes

### PR DESCRIPTION
**Pull request by RensDofferhoff**
- Update R version.
- Fix developer module loading error

### CODE REVIEW

**Comment by RensDofferhoff** [comment]
Contains https://github.com/jasp-stats/jasp-desktop/pull/5096 but on development
Along with cmake fixes and fixes to get dev-modules working again.
Fixes heplots issue for development

**Review by JorisGoosen — APPROVED** [review]

**Comment by JorisGoosen** [comment]
With the commit added it works for me on mac arm
I cant test on windows or linux atm

File: Modules/install-module.R.in [hunk 1]
```diff
@@ -30,6 +30,12 @@ options(
   configure.vars = c(jaspBase = "INCLUDE_DIR='@PROJECT_SOURCE_DIR@/Common'") #Needed for flatpak build as it keeps recompiling jaspBase (which it shouldnt of course but I dont know why) and this is an easy fix to get it to work now
 )
 
+if (jaspBase::getOS() == "osx") {
+	options(pkgType = "mac.binary")
+} else if (jaspBase::getOS() == "windows") {
+	options(pkgType = "win.binary")
+}
```
Comment (vandenman) [inline Modules/install-module.R.in]:
```suggestion
if (jaspBase::getOS() == "osx") {
  options(pkgType = "mac.binary")
} else if (jaspBase::getOS() == "windows") {
  options(pkgType = "win.binary")
}
```

Comment (vandenman) [reply Modules/install-module.R.in]: maybe we should just make a .vscode-workspace file that sets all these things correctly... not sure if something like that exists for QtCreator though (it exists for C++/ QML but I'm not sure about custom file formats).

**Review by vandenman — APPROVED** [review]

**Pull request merged**